src/braket/experimental/autoqasm/types/conversions.py

```diff
@@ -37,6 +38,10 @@ def map_parameter_type(python_type: type) -> type:
     """
     origin_type = typing.get_origin(python_type) or python_type
 
+    # todo see if we can reuse `is_qubit_identifier_type` or at least
```

Comment:
We should avoid circular import if we move `is_qubit_identifier_type` to `types` module. The method is more of a type method than a qubit method anyway. 

However, `is_qubit_identifier_type` checks the type of instantiated object. Here, I think we can use `QubitIdentifierType` directly with `python_type is QubitIdentifierType`.

Reply:
Done, works!

Reply:
thanks for refactoring!